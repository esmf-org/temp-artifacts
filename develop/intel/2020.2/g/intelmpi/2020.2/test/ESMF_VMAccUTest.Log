2022-07-20 00:34:47
20220720 004347.482 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.482 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 004347.482 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 004347.482 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 004347.482 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 004347.482 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.482 INFO             PET0 Running with ESMF Version   : v8.4.0b06-4-gd14c497
20220720 004347.482 INFO             PET0 ESMF library build date/time: "Jul 20 2022" "00:33:16"
20220720 004347.482 INFO             PET0 ESMF library build location : /work/noaa/nems/tufuk/esmf-test-scripts/python_scripts/test/intel_2020.2_intelmpi_g_develop/esmf
20220720 004347.482 INFO             PET0 ESMF_COMM                   : intelmpi
20220720 004347.483 INFO             PET0 ESMF_MOAB                   : enabled
20220720 004347.483 INFO             PET0 ESMF_LAPACK                 : enabled
20220720 004347.483 INFO             PET0 ESMF_NETCDF                 : enabled
20220720 004347.483 INFO             PET0 ESMF_PNETCDF                : disabled
20220720 004347.483 INFO             PET0 ESMF_PIO                    : enabled
20220720 004347.483 INFO             PET0 ESMF_YAMLCPP                : enabled
20220720 004347.535 INFO             PET0  Beginning Test, file ESMF_VMAccUTest.F90, line 52
20220720 004347.535 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220720 004347.535 INFO             PET0  Number of accelerator devices =           0
20220720 004347.536 INFO             PET0  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, line 72
20220720 004347.536 INFO             PET0  Ending Test, file ESMF_VMAccUTest.F90, line 76
20220720 004347.536 INFO             PET0 Finalizing ESMF
20220720 004347.494 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.494 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 004347.494 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 004347.494 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 004347.494 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 004347.494 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.494 INFO             PET1 Running with ESMF Version   : v8.4.0b06-4-gd14c497
20220720 004347.494 INFO             PET1 ESMF library build date/time: "Jul 20 2022" "00:33:16"
20220720 004347.494 INFO             PET1 ESMF library build location : /work/noaa/nems/tufuk/esmf-test-scripts/python_scripts/test/intel_2020.2_intelmpi_g_develop/esmf
20220720 004347.494 INFO             PET1 ESMF_COMM                   : intelmpi
20220720 004347.497 INFO             PET1 ESMF_MOAB                   : enabled
20220720 004347.497 INFO             PET1 ESMF_LAPACK                 : enabled
20220720 004347.497 INFO             PET1 ESMF_NETCDF                 : enabled
20220720 004347.497 INFO             PET1 ESMF_PNETCDF                : disabled
20220720 004347.497 INFO             PET1 ESMF_PIO                    : enabled
20220720 004347.497 INFO             PET1 ESMF_YAMLCPP                : enabled
20220720 004347.548 INFO             PET1  Beginning Test, file ESMF_VMAccUTest.F90, line 52
20220720 004347.548 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220720 004347.548 INFO             PET1  Number of accelerator devices =           0
20220720 004347.548 INFO             PET1  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, line 72
20220720 004347.548 INFO             PET1  Ending Test, file ESMF_VMAccUTest.F90, line 76
20220720 004347.548 INFO             PET1 Finalizing ESMF
20220720 004347.496 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.496 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 004347.496 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 004347.496 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 004347.496 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 004347.496 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.496 INFO             PET2 Running with ESMF Version   : v8.4.0b06-4-gd14c497
20220720 004347.496 INFO             PET2 ESMF library build date/time: "Jul 20 2022" "00:33:16"
20220720 004347.496 INFO             PET2 ESMF library build location : /work/noaa/nems/tufuk/esmf-test-scripts/python_scripts/test/intel_2020.2_intelmpi_g_develop/esmf
20220720 004347.496 INFO             PET2 ESMF_COMM                   : intelmpi
20220720 004347.497 INFO             PET2 ESMF_MOAB                   : enabled
20220720 004347.497 INFO             PET2 ESMF_LAPACK                 : enabled
20220720 004347.497 INFO             PET2 ESMF_NETCDF                 : enabled
20220720 004347.497 INFO             PET2 ESMF_PNETCDF                : disabled
20220720 004347.497 INFO             PET2 ESMF_PIO                    : enabled
20220720 004347.497 INFO             PET2 ESMF_YAMLCPP                : enabled
20220720 004347.548 INFO             PET2  Beginning Test, file ESMF_VMAccUTest.F90, line 52
20220720 004347.548 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220720 004347.548 INFO             PET2  Number of accelerator devices =           0
20220720 004347.549 INFO             PET2  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, line 72
20220720 004347.549 INFO             PET2  Ending Test, file ESMF_VMAccUTest.F90, line 76
20220720 004347.549 INFO             PET2 Finalizing ESMF
20220720 004347.496 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.496 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 004347.496 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 004347.496 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 004347.496 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 004347.496 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 004347.496 INFO             PET3 Running with ESMF Version   : v8.4.0b06-4-gd14c497
20220720 004347.496 INFO             PET3 ESMF library build date/time: "Jul 20 2022" "00:33:16"
20220720 004347.496 INFO             PET3 ESMF library build location : /work/noaa/nems/tufuk/esmf-test-scripts/python_scripts/test/intel_2020.2_intelmpi_g_develop/esmf
20220720 004347.496 INFO             PET3 ESMF_COMM                   : intelmpi
20220720 004347.509 INFO             PET3 ESMF_MOAB                   : enabled
20220720 004347.509 INFO             PET3 ESMF_LAPACK                 : enabled
20220720 004347.509 INFO             PET3 ESMF_NETCDF                 : enabled
20220720 004347.509 INFO             PET3 ESMF_PNETCDF                : disabled
20220720 004347.509 INFO             PET3 ESMF_PIO                    : enabled
20220720 004347.509 INFO             PET3 ESMF_YAMLCPP                : enabled
20220720 004347.561 INFO             PET3  Beginning Test, file ESMF_VMAccUTest.F90, line 52
20220720 004347.561 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220720 004347.561 INFO             PET3  Number of accelerator devices =           0
20220720 004347.561 INFO             PET3  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, line 72
20220720 004347.561 INFO             PET3  Ending Test, file ESMF_VMAccUTest.F90, line 76
20220720 004347.561 INFO             PET3 Finalizing ESMF
