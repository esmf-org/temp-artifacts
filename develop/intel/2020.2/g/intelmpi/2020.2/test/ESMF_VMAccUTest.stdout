2022-07-20 00:34:47
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
  Beginning Test, file ESMF_VMAccUTest.F90, line 52
  NUMBER_OF_PROCESSORS           4
  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, lin
 e 72
  Ending Test, file ESMF_VMAccUTest.F90, line 76
  Beginning Test, file ESMF_VMAccUTest.F90, line 52
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_VMAccUTest.F90, line 52
  NUMBER_OF_PROCESSORS           4
  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, lin
 e 72
  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, lin
  Ending Test, file ESMF_VMAccUTest.F90, line 76
 e 72
  Ending Test, file ESMF_VMAccUTest.F90, line 76
  Beginning Test, file ESMF_VMAccUTest.F90, line 52
  NUMBER_OF_PROCESSORS           4
  PASS  Test getting the number of accelerator devices, ESMF_VMAccUTest.F90, lin
 e 72
  Ending Test, file ESMF_VMAccUTest.F90, line 76
  PET           1  Test Elapsed Time    43.3310000000000       msec.
  PET           2  Test Elapsed Time    44.8000000000000       msec.
  PET           3  Test Elapsed Time    43.9240000000000       msec.
  PET           0  Test Elapsed Time    54.9060000000000       msec.
